qa/L0_http/generate_endpoint_test.py

```diff
@@ -142,6 +142,49 @@ def test_generate(self):
         self.assertIn("TEXT", data)
         self.assertEqual(text, data["TEXT"])
 
+    def test_request_id(self):
```

Comment:
Thanks for adding the test! I will run a pipeline with these changes.

Reply:
Pipeline looks mostly good, however, please update this variable from 15 to 16 to account for the new test added: https://github.com/triton-inference-server/server/blob/e0d80d46966bdf83d5dedf5b76f08e0aa40607ec/qa/L0_http/test.sh#L665

```
EXPECTED_NUM_TESTS=16
```

Reply:
Thanks. Updated count to 16

Reply:
Thanks! Ran a new pipeline 16473896 and it passed :rocket: Once I verify the CLA, this can be merged :+1: